bugToJira.py

```diff
@@ -0,0 +1,221 @@
+#!/bin/python3
+
+import getpass
+import os
+from jira import JIRA
+import mysql.connector
+
+ANDROID_VERSION_KEY = '[ANDROID_VERSION_KEY]'
+DESCRIPTION_KEY = '[DESCRIPTION_KEY]'
+JIRA_LABEL = 'From_Bugzilla'
+
+# Functions!
+
+# Get a value from the user
```

Comment:
The common use in Python is to put this kind of comment after the method signature, surrounded by `"""`. Look at here for an example.